app/graphql/types/event_type.rb

```diff
@@ -1,17 +1,38 @@
 module Types
   class EventType < Types::BaseObject
+    description 'An event represents a date in time that a partner operates their service, meetup, workout or class etc'
+
+    field :id, ID, 
+      null: false,
+      description: 'Our internal ID for further querying'
 
-    field :id, ID, null: false
     # Summary and name are aliases, this is left as a convenience 
     #   for people used to iCal format
-    field :name, String, method: :summary
-    field :summary, String
-    field :description, String
-    field :startDate, String, method: :dtstart, null: false
-    field :endDate, String, method: :dtend
-
-    field :address, AddressType
-    field :organizer, PartnerType, method: :partner
+    field :name, String, 
+      method: :summary,
+      description: 'An alias for `summary`'
+
+    field :summary, String,
+      description: 'A description of this specific event'
```

Comment:
The title of the event

Reply:
which one do you mean that description for `title` or `summary` ?